cluster-autoscaler/cloudprovider/gce/autoscaling_gke_client_v1beta1_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gce
+
+import (
+	"net/http"
+	"testing"
+
+	test_util "k8s.io/autoscaler/cluster-autoscaler/utils/test"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	gke_api_beta "google.golang.org/api/container/v1beta1"
+)
+
+const operationRunningResponse = `{
+  "name": "operation-1505728466148-d16f5197",
+  "zone": "us-central1-a",
+  "operationType": "CREATE_NODE_POOL",
+  "status": "RUNNING",
+  "selfLink": "https://container.googleapis.com/v1/projects/601024681890/locations/us-central1-a/operations/operation-1505728466148-d16f5197",
+  "targetLink": "https://container.googleapis.com/v1/projects/601024681890/locations/us-central1-a/clusters/cluster-1/nodePools/nodeautoprovisioning-323233232",
+  "startTime": "2017-09-18T09:54:26.148507311Z",
+  "endTime": "2017-09-18T09:54:35.124878859Z"
+}`
+
+const operationDoneResponse = `{
+  "name": "operation-1505728466148-d16f5197",
+  "zone": "us-central1-a",
+  "operationType": "CREATE_NODE_POOL",
+  "status": "DONE",
+  "selfLink": "https://container.googleapis.com/v1/projects/601024681890/locations/us-central1-a/operations/operation-1505728466148-d16f5197",
+  "targetLink": "https://container.googleapis.com/v1/projects/601024681890/locations/us-central1-a/clusters/cluster-1/nodePools/nodeautoprovisioning-323233232",
+  "startTime": "2017-09-18T09:54:26.148507311Z",
+  "endTime": "2017-09-18T09:54:35.124878859Z"
+}`
+
+func newTestAutoscalingGkeClientV1beta1(t *testing.T, project, location, clusterName, url string) *autoscalingGkeClientV1beta1 {
```

Comment:
Shouldn't you override operation timeout & poll interval somewhere in this file?

Reply:
Done